### PR TITLE
Handoff - Show atributes

### PR DESCRIPTION
**Pull request by Alotor**

### CODE REVIEW

File: frontend/src/app/main/data/viewer.cljs [hunk 1]
```diff
@@ -236,16 +243,92 @@
...
+            selection (-> state
+                          (get-in [:viewer-local :selected] #{})
+                          (conj id))]
+        (-> state
+            (assoc-in [:viewer-local :selected]
+                      (cph/expand-region-selection objects selection)))))))
+
+(defn select-all
+  []
+  (ptk/reify ::shift-select-to
```
Comment (hirunatan) [inline frontend/src/app/main/data/viewer.cljs]: change keyword

File: frontend/src/app/main/ui/viewer/handoff/attributes/text.cljs [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        color (shape->color style)
+        to-prop {:fill-color "color"
+                 :fill-color-gradient "color"}]
+    [:div.attributes-text-block
+     [:div.attributes-typography-row
+      [:div.typography-sample
+       {:style {:font-family (:font-family full-style)
+                :font-weight (:font-weight full-style)
+                :font-style (:font-style full-style)}}
+       (t locale "workspace.assets.typography.sample")]
```
Comment (hirunatan) [inline frontend/src/app/main/ui/viewer/handoff/attributes/text.cljs:65]: I think this text is not in figma prototype. And in locales.json is just "Ag" ¿?

File: frontend/src/app/main/data/viewer.cljs (hunk 1 above)
Comment (Alotor) [reply frontend/src/app/main/data/viewer.cljs]: ok

File: frontend/src/app/main/ui/viewer/handoff/attributes/text.cljs (hunk 2 above)
Comment (Alotor) [reply frontend/src/app/main/ui/viewer/handoff/attributes/text.cljs:65]: Yes it is

**Pull request merged**